internal/generate/testdata/go/deploy/olm-catalog/memcached-operator/noupdate/memcached-operator.v0.0.3.clusterserviceversion.yaml

```diff
@@ -193,7 +193,7 @@ spec:
   - admissionReviewVersions: null
     deploymentName: memcached-operator-webhook
     failurePolicy: Fail
-    name: mmemcached.kb.io
+    generateName: mmemcached.kb.io
     rules:
     - apiGroups:
       - cache.example.com
```

Comment:
Shows that it is missing updated the content of all CSV in : operator-sdk/internal/scorecard/alpha/testdata/